src/components/Filterbar/FilterBar.jsx

```diff
@@ -205,44 +204,42 @@ import { useProduct } from '../../context/product-context';
           <input
             type="radio"
             value="4Starts"
-            name="rating" 
-            checked={sortByRatings==="FOUR_AND_ABOVE"}
-            onChange={()=>dispatch({type:"SORT_BY_RATING_4"})}           
+            name="rating"
+            checked={sortByRatings === "FOUR_AND_ABOVE"}
+            onChange={() => dispatch({ type: "SORT_BY_RATING_4" })}
           />
-          4 <FaRegStar/> & above
+          4 <FaRegStar /> & above
         </li>
         <li className="sidebar-list">
           <input
             type="radio"
             value="3Starts"
-            name="rating"    
-            checked={sortByRatings==="THREE_AND_ABOVE"}
-            onChange={()=>dispatch({type:"SORT_BY_RATING_3"})}         
+            name="rating"
+            checked={sortByRatings === "THREE_AND_ABOVE"}
+            onChange={() => dispatch({ type: "SORT_BY_RATING_3" })}
           />
-          3 <FaRegStar/> & above
+          3 <FaRegStar /> & above
         </li>
         <li className="sidebar-list">
           <input
             type="radio"
             value="2starts"
-            name="rating"   
-            checked={sortByRatings==="TWO_AND_ABOVE"}
-            onChange={()=>dispatch({type:"SORT_BY_RATING_2"})}          
+            name="rating"
+            checked={sortByRatings === "TWO_AND_ABOVE"}
+            onChange={() => dispatch({ type: "SORT_BY_RATING_2" })}
           />
-          2 <FaRegStar/> & above
+          2 <FaRegStar /> & above
         </li>
         <li className="sidebar-list">
```

Comment:
Instead of writing multiple `action types` you could have pass the `rating value` in payload. In this way you will have only one `action type` and one `filter` function.

Reply:
will try to do